Sources/Gifu/Classes/Animator.swift

```diff
@@ -136,9 +145,12 @@ public class Animator {
   /// - parameter size: The target size of the individual frames.
   /// - parameter contentMode: The view content mode to use for the individual frames.
   /// - parameter loopCount: Desired number of loops, <= 0 for infinite loop.
-  /// - parameter completionHandler: Completion callback function
```

Comment:
Note that the parameter name here was outdated, I changed it along with a more descriptive ... description :)